packages/roosterjs-editor-api/lib/format/toggleBlockQuote.ts

```diff
@@ -0,0 +1,91 @@
+import execFormatWithUndo from './execFormatWithUndo';
```

Comment:
Select multiple lines and blockquote, only the first line is quoted.

Reply:
Fixed